operator-k8s/src/config.rs

```diff
@@ -1,13 +1,10 @@
 use clap::Parser;
 
-/// Deploy operator config
+/// Xline operator config
 #[derive(Debug, Parser)]
 #[non_exhaustive]
 #[command(author, version, about, long_about = None)]
 pub struct Config {
-    /// The address on which the heartbeat HTTP server will listen to
-    #[arg(long, default_value = "0.0.0.0:8080")]
-    pub listen_addr: String,
     /// The namespace to deploy
     #[arg(long, default_value = "default")]
     pub namespace: String,
```

Comment:
Can we merge `namespace` and `cluster_wide` into one enum like this:
```rust
enum NameSpace {
    Single(String),
    ClusterWide
}
```

Reply:
This is indeed a good improvement, but I'm unsure about how to properly parse this enum. The argument following `--namespace` should be a string, representing a namespace. The `cluster_wide` indicates no namespace constraints, but how should we denote cluster_wide?
Perhaps the behavior of using the `default` namespace by default could be modified to set the `cluster_wide` by default.
What is your opinion?

Reply:
LGTM